src/webview_provider/newDocsGenPanel.ts

```
@@ -196,7 +196,7 @@ export class NewDocsGenPanel
         }
 
         const { documentation, message: missingDocumentationMessage } =
-          await this.docGenService.getDocumentationForCurrentActiveFile();
+          await this.docGenService.getUncompiledDocumentationForCurrentActiveFile();
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)